### PR TITLE
Tbolt/3449 Updates to default messages for empty fields in export view

### LINKED ISSUE #3449: [Feature] Update default messages for empty fields in export view
Design task #3345

**Description and related issues -**
Some of the fields in the eAPD will not be required to be answered before submit. We have language that we use, noted in the content guide in the wiki, to note when a field isn't filled out so that it's clear to the reviewer that it wasn't a mistake or something broken in the system, but we only did this for top fields before release. Update the other fields to have default values.

### Activity
* Provide a short overview of summary
```
No response was provided.
```
* Statement of Alternative Considerations and Supporting Justification
```
No response was provided. 
```
* Outcomes and Metrics
  * Use `No outcome(s) and corresponding metrics were provided.` when the section is missing at least one outcome and metrics
  * Use `No metrics were specified for this outcome.` when an outcome is included but no metrics.
* Milestones
  * Use `No milestones were provided.` for an empty milestone section
* State Staff
```
State staff not specified.
```
* Other state expense
```
Other state expenses not specified.
```
* Private Contractor
```
Private contractor(s) not specified.
```
* Description of Cost Allocation Methodology
```
No cost allocation methodology was provided.
```
* Other funding description
```
No response was provided.
```
* Budget and Federal Financial Participation (FFP)
```
No response was provided for the Budget and FFP.
```

### Activity Schedule Summary

* Activity milestones
  * Use `Activity X: Untitled | No milestones were provided` if the activity is unnamed and there are no milestones.
  * Use `Activity X: Untitled Milestones` if there are milestones but the activity is not unnamed.
  * Use `No milestones to display.` in the table cell if there are no milestones.
  * Use `Dates not specified` when the Activity Date Range is not available.
  * Use `Milestone name not specified.` if a milestone is unnamed. 

### Assurances and Compliance

* Each field would have no response was provided. i.e.: 
```
Does this project comply with 42 CFR Part 495.348?
No response was provided.
```

**Mockups or link to Figma**
https://www.figma.com/file/6eVvo7JjvXiovGTR4BoVgK/CMS-eAPD-2021?node-id=3524%3A0

https://www.figma.com/file/6eVvo7JjvXiovGTR4BoVgK/CMS-eAPD-2021?node-id=4741%3A31123

https://www.figma.com/file/6eVvo7JjvXiovGTR4BoVgK/CMS-eAPD-2021?node-id=4743%3A31165

### This task is done when…

- [ ] all of the above default values have been updated
- [ ] all tests, including e2e tests, have been updated to match the changes

### PR DESCRIPTION
**Pull request by tbolt**
Resolves #3449 

**Description-**
This PR updates or adds the default messages for when fields are empty in the export view. See original ticket for specifics.

**Steps to manually verify this change...**

1. Login to view/export a APD
2. Compare the 3449 ticket language with new update messages

### This pull request is ready to review when...

- [x] Automated tests are updated (and all tests are passing)
- [ ] The change has been documented
- [ ] Associated OpenAPI documentation has been updated
- [ ] Changelog is updated as appropriate
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented

### This pull request can be merged when…

- [ ] Code has been reviewed by someone other than the original author
- [ ] QA has verified the accessibility and functionality related to the change
- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: web/src/pages/apd/activities/activities-dashboard/ActivityReadOnly.js [hunk 1]
```diff
@@ -221,10 +224,12 @@ const Activity = ({ activity, activityIndex }) => {
         <br />
         Outcomes and Metrics
       </h3>
+      {activity.outcomes.length === 0 && 'No outcome(s) and corresponding metrics were provided.'}
```
Comment (thetif) [inline web/src/pages/apd/activities/activities-dashboard/ActivityReadOnly.js:227]: shouldn't this be
```
No outcome(s) or corresponding metrics...
```

**Review by thetif — APPROVED** [review]
looks good, one nit-picky grammar thing

**Review by jeromeleecms — CHANGES_REQUESTED** [review]
Outcome with no metric doesn't show the language "No metrics were specified for this outcome."

image

Also, Per @thetif comments, I would be OK with updating this language:

"No outcome(s) and corresponding metrics were provided."

to 

"No outcome(s) and/or corresponding metric(s) were provided."

**Comment by tbolt** [comment]
@jeromeleecms Good catch, that might have been reverted in the merge. I will update the language as well.

**Review by beparticular — APPROVED** [review]
The only comment I had was also what Jerome and Tif have fixed. Approved

**Comment by tbolt** [comment]
@jeromeleecms Thanks, take a look now it should all be working

**Comment by tbolt** [comment]
@jeromeleecms ok _now_ it should be working. I just tested the preview build and it was updated and working but let me know if you run into any other issues.

**Comment by mirano-darren** [comment]
Nice work!
One question, and I guess this one is more for #3345, but where is the description for Budget and FFP come from? As of now it's just a blank section if it's default:
image
The ticket says to have the default message as `No response was provided for the Budget and FFP`. However there is no text boxes on the Budget and FFP page?

**Review by mirano-darren — APPROVED** [review]
😄

**Review by itsonlydio — COMMENTED** [review]
Looks good to me!

**Pull request merged**